src/Config.h

```diff
@@ -14,6 +14,7 @@ class Config
                                              "VRAM", "GPU",      "RAM",   "CPU",       "Battery", "Power"};
 
     std::string cpuThermalZone = "";     // idk, no standard way of doing this.
+    std::string gpuThermalZone = "";
```

Comment:
Please put the default value (The old one, same as comment in module.nix) here (Also for backwards compatibility)